src/main.cpp

```diff
@@ -1,29 +1,32 @@
 #include <Arduino.h>
```

Comment:
Let's move this to `hardware/tests/hw_test_motors.cpp`

Reply:
yea good idea